release-notes.html.md.erb

```diff
@@ -189,7 +189,7 @@ This release has the following breaking changes:
 
 This release has the following issues:
 
- -
+ - Pushing apps or retrieving app logs with cf logs requires cf CLI v7.0.1 or later
```

Comment:
could we put "cf logs" in back-ticks like `cf logs` so that it's clearer that it's a command?

Reply:
Done. Also fixed some formatting issues in Release Notes.